tests/test_api_token.py

```diff
@@ -126,6 +127,88 @@ def test_01_test_two_tokens(self):
             # we have two tokens
             self.assertEqual(0, result.get("value").get("count"))
 
+    def test_02_two_resolver_in_realm_policy_condition(self):
+        self.setUp_user_realms()
+        # add a second resolver to the realm
+        save_resolver({"resolver": self.resolvername2,
+                       "type": "passwdresolver",
+                       "fileName": PWFILE2})
+        added, failed = set_realm(self.realm1,
+                                  [self.resolvername2])
+        self.assertEquals(len(failed), 0)
+        self.assertEquals(len(added), 1)
+
+        # create token delete policy for "testadmin" on resolver1
+        set_policy(name="pol-reso1",
+                   scope=SCOPE.ADMIN,
+                   action=','.join([ACTION.DELETE, ACTION.ASSIGN, ACTION.UNASSIGN,
+                                    ACTION.DISABLE, ACTION.ENABLE]),
+                   user=self.testadmin,
```

Comment:
I think this is wrong. This policy matches only for the actions on the user who is called "self.testadmin".
You probably want to use the parameter "adminuser" of `set_policy`.
So I assume the following tests *could* also fail, since the user name does not match.

Please consider also adding a positive test.

Reply:
Correct, with the positive tests this makes sense now.
Fixed in 81cf2f74